src/rgw/rgw_rados.cc

```diff
@@ -2446,15 +2446,6 @@ int RGWRados::Bucket::List::list_objects_ordered(
 
   string skip_after_delim;
   while (truncated && count <= max) {
-    if (skip_after_delim > cur_marker.name) {
-      cur_marker = skip_after_delim;
-
-      ldout(cct, 20) << "setting cur_marker="
-		     << cur_marker.name
-		     << "[" << cur_marker.instance << "]"
-		     << dendl;
-    }
-    std::map<string, rgw_bucket_dir_entry> ent_map;
```

Comment:
probably we should keep the definition of `ent_map`.

Reply:
indeed! fixing.